pallets/nfts/src/benchmarking.rs

```diff
@@ -274,6 +274,7 @@ benchmarks_instance_pallet! {
 	mint {
 		let (collection, caller, caller_lookup) = create_collection::<T, I>();
 		let item = T::Helper::item(0);
+		T::Currency::make_free_balance_be(&caller, T::Currency::minimum_balance() + T::CollectionDeposit::get() + T::BalanceDeposit::get());
```

Comment:
Note: This overrides the balance of `caller` in `create_collection`. The purpose is to make this more specific to what are reserved in the function.